pom.xml

```diff
@@ -182,7 +182,7 @@
     <spark.bundle.hive.shade.prefix/>
     <utilities.bundle.hive.scope>provided</utilities.bundle.hive.scope>
     <utilities.bundle.hive.shade.prefix/>
-    <argLine>-Xmx2g</argLine>
+    <argLine>-Xms4g -Xmx4g</argLine>
```

Comment:
I will restore this config and try to run again.

Previously it may lead JVM for maven surefire to crash with no specific reason. Incresing the JVM heap memory can alleviate this.